### PR TITLE
Feature/1160 add swagger

### PR DESCRIPTION
**Pull request by jauschalley**
*What does this PR do?*

*Jira ticket number?*
[ISPGBSS-](https://jiraent.cms.gov/browse/ISPGBSS-1160)

*Changes*

- Adding swagger (with and without ui) to our api

*Testing*

- [ ] Step 1 : Go to http://localhost:8000/doc/ to see the SwaggerUI
- [ ] Step 2: http://localhost:8000/doc.json or http://localhost:8000/doc.yaml
 to see the unformatted spec

### CODE REVIEW

File: blueprintapi/urls.py [hunk 1]
```diff
@@ -14,10 +14,23 @@
...
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
 
 from . import views
 
+schema_view = get_schema_view(
+    openapi.Info(
+        title="RapidATO API",
+        default_version='v1',
+        description="Welcome to the world of RapidATO",
```
Comment (jwhite3) [inline blueprintapi/urls.py:28]: For some reason, my brain stopped reading this sentence part way through and changed the sentence to "Welcome... to Jurassic Park"

File: blueprintapi/views.py [hunk 2]
```diff
@@ -2,7 +2,6 @@
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
 
-
```
Comment (jwhite3) [inline blueprintapi/views.py:5]: Why'd you remove this space? Two blank spaces before a function or class definition is actually preferred: https://peps.python.org/pep-0008/#blank-lines

**Review by jwhite3 — DISMISSED** [review]

**Review by amyd11 — DISMISSED** [review]
Non-blocking suggestion:

This seems like something worth adding to the readme:
Go to http://localhost:8000/doc/ to see the SwaggerUI
Go to http://localhost:8000/doc.json or http://localhost:8000/doc.yaml to see the unformatted spec

File: blueprintapi/views.py (hunk 2 above)
Comment (jauschalley) [reply blueprintapi/views.py:5]: Ah, sloppy removal of a comment.  Thanks for the catch.

**Comment by jauschalley** [comment]
Good idea! @amyd11

**Review by jwhite3 — DISMISSED** [review]

**Pull request closed without merging**